src/main/java/seedu/address/logic/commands/recipe/AddRecipeCommand.java

```diff
@@ -34,7 +34,7 @@
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "Chicken salad "
-            + PREFIX_INGREDIENT + "chicken - 100g, lettuce - a bit, tomato "
+            + PREFIX_INGREDIENT + "chicken - 100.50g, lettuce - a bit, tomato -3/4 cups"
```

Comment:
in the first quantity, there a space "-(space)100.5g", but in the 2nd one there isnt "-3/4 cups", maybe can standardize?

Reply:
ok